src/main/java/com/jherrick/ConsolePlayerAgent.java

```diff
@@ -33,4 +47,23 @@ public Move pickMove(Board b) {
             }
         }
     }
+
+    
+    // ================ Helper Functions =================
+
+    private boolean moveStringFormatCheck(String s){
```

Comment:
I usually name methods that return booleans along the lines of is____, has____, should____. So then when you're reading the code, the intended directionality is clear 

if isValidMove(moveString), then makeMove 
reads a lot better than 
if moveFormatCheck(s), then makeMove